common/supplemental/languageInfo.xml

```diff
@@ -96,7 +96,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<languageMatch desired="mai"	supported="hi"	distance="20"	oneway="true"/>	<!-- Maithili ⇒ Hindi -->
 			<languageMatch desired="mfe"	supported="en"	distance="30"	oneway="true"/>	<!-- Morisyen: mfe ⇒ en -->
 			<languageMatch desired="mg"	supported="fr"	distance="30"	oneway="true"/>	<!-- Malagasy: mg ⇒ fr -->
-			<languageMatch desired="mi"	supported="en"	distance="20"	oneway="true"/>	<!-- Maori: mi ⇒ en -->
+			<languageMatch desired="mi"	supported="en"	distance="20"	oneway="true"/>	<!-- Māori: mi ⇒ en -->
```

Comment:
Cases like these in comments won't matter (and will typically be regenerated when the tools are run later on).

But they don't hurt either.